server/transaction/manager.go

```diff
@@ -79,18 +77,20 @@ func (c *SessionCtx) GetStagedDatabase() interface{} {
 // this will create a session tracker for tracking the sessions.
 
 type Manager struct {
-	kvStore kv.KeyValueStore
+	kvStore  kv.KeyValueStore
+	chunking bool
 }
 
 func NewManager(kvStore kv.KeyValueStore) *Manager {
 	return &Manager{
-		kvStore: kvStore,
+		kvStore:  kvStore,
+		chunking: false,
```

Comment:
I think we should make this a config option so we can run it with our tests and on dev

Reply:
that's the plan, we need to pass in the manager - that will be part of follow up diff when I'll hook it up with the API layer.